src/main/java/CollectingCharactersDataViaTerminal.java

```diff
@@ -0,0 +1,12 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class CollectingCharactersDataViaTerminal {
+    List<String> charactersNamesList = new ArrayList<>();
+    Scanner namesInput = new Scanner(System.in);
+
+    List<Integer> charactersWeightsList = new ArrayList<>();
+    Scanner weightsInput = new Scanner(System.in);
```

Comment:
Zrobienie 2 inputów `namesInput` i `weightsInput` nic Ci nie daje. Obiektu stworzonego przez `new Scanner(System.in);` możesz użyć do pobrania dowolnego typu rzeczy z consoli, nie trzeba tworzyć go unikatowo dla każdej pobieranej wartości, ponieważ sposób pobierania wartości będzie taki sam dla każdej tej wartości.

Nie trzymałbym też tych scannerów w tej klasie, ponieważ to ogranicza jej możliwości, bo uzależniamy ją do skanera. Jakbym chciał tej klasy użyć w przyszłości do aplikacji webowej (gdzie pojęcie takie jak Scanner nie występuje* i tutaj gwiazdka ale to nie istotne) to mam teraz problem, bo muszę ciągnąć ze sobą ten scanner.